pandas/tests/extension/base/reduce.py

```diff
@@ -25,6 +25,12 @@ class BaseNoReduceTests(BaseReduceTests):
 
     @pytest.mark.parametrize("skipna", [True, False])
     def test_reduce_series_numeric(self, data, all_numeric_reductions, skipna):
+        if isinstance(data, pd.arrays.StringArray) and all_numeric_reductions in [
+            "min",
+            "max",
+        ]:
+            pytest.skip("These reductions are implemented")
```

Comment:
Can you see if you can rather update this in `test_string.py` ? It might be we now need to subclass the ReduceTests instead of NoReduceTests. 
(ideally the base tests remain dtype agnostic)

Reply:
By updating in `test_string.py` do you mean adding tests using the fixtures `data` and `all_numeric_reductions`, only checking for the "correct" output (and skipping over those reductions that aren't yet implemented)?

Reply:
Hmm, actually looking at the base reduction tests now: they are not really written in a way that they will pass for strings. 

But so you can copy this test to `tests/extension/test_strings.py` (and so override the base one), and then do the string-array-specific adaptation there.  It gives some duplication of the test code, but it's not long, and it clearer separation of concerns (the changes for string array are in test_string)

Reply:
Ok, so we can remove the special cases for `StringArray` in `BaseNoReduceTests` without getting test failures, as long as they're handled in `TestNoReduce` in `test_string.py`? I'm not too familiar with how these particular tests actually get executed during CI